code/modules/wod13/items/clothes/suit.dm

```diff
@@ -505,4 +505,383 @@
 	icon_state = "pentex_labcoat_alt"
 	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, RAD = 0, FIRE = 0, ACID = 90, WOUND = 10)
 
+/obj/item/clothing/suit/vampire/greatcoat
```

Comment:
```suggestion
/obj/item/clothing/suit/vampire/greatcoat // APOC EDIT START
```
modular comment. if the codebase wasn't dead i'd ask this to be in a modular folder but whatever